server/routes/induction/common/additionalTrainingController.ts

```diff
@@ -17,7 +17,7 @@ export default abstract class AdditionalTrainingController extends InductionCont
     res: Response,
     next: NextFunction,
   ): Promise<void> => {
-    const { inductionDto } = req.session
+    const inductionDto = req.session.inductionDto ?? req.journeyData?.inductionDto
```

Comment:
These controllers in `induction/common` are used for both the Create and Update journey
In this PR we are changing the Update journey to store and reference the DTO in journeyData, but the Create journey still uses the session; hence this logic in each controller.

(soon we will migrate the Create Induction journey to use the journeyData approach, so this code will be simplified again at that point)